dotnet/src/Agents/Abstractions/AgentChat.cs

```diff
@@ -17,12 +18,15 @@ namespace Microsoft.SemanticKernel.Agents;
 /// Any <see cref="AgentChat" /> instance does not support concurrent invocation and
 /// will throw exception if concurrent activity is attempted for any public method.
 /// </remarks>
-public abstract class AgentChat
+/// <remarks>
+/// Initializes a new instance of the <see cref="AgentChat"/> class.
+/// </remarks>
+public abstract class AgentChat(ILogger logger)
 {
-    private readonly BroadcastQueue _broadcastQueue;
-    private readonly Dictionary<string, AgentChannel> _agentChannels; // Map channel hash to channel: one entry per channel.
-    private readonly Dictionary<Agent, string> _channelMap; // Map agent to its channel-hash: one entry per agent.
-
+    private readonly BroadcastQueue _broadcastQueue = new();
+    private readonly Dictionary<string, AgentChannel> _agentChannels = []; // Map channel hash to channel: one entry per channel.
+    private readonly Dictionary<Agent, string> _channelMap = []; // Map agent to its channel-hash: one entry per agent.
+    private readonly ILogger _logger = logger;
```

Comment:
What happenend to logger-factory? 

![image](https://github.com/microsoft/semantic-kernel/assets/66376200/5aedb6f8-2f45-4756-98b1-6896ae521ef2)